opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -245,84 +137,161 @@ message MetricDescriptor {
-    // CUMULATIVE metric:
-    //
-    //   1. The system starts receiving at time=t_0.
-    //   2. A request is received, the system measures 1 request.
-    //   3. A request is received, the system measures 1 request.
-    //   4. A request is received, the system measures 1 request.
-    //   5. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_0 to
-    //      t_0+1 with a value of 3.
-    //   6. A request is received, the system measures 1 request.
-    //   7. A request is received, the system measures 1 request.
-    //   8. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_0 to
-    //      t_0+2 with a value of 5.
-    //   9. The system experiences a fault and loses state.
-    //   10. The system recovers and resumes receiving at time=t_1.
-    //   11. A request is received, the system measures 1 request.
-    //   12. The 1 second collection cycle ends. A metric is exported for the
-    //      number of requests received over the interval of time t_1 to
-    //      t_0+1 with a value of 1.
-    //
-    // Note: Even though, when reporting changes since last report time, using 
-    // CUMULATIVE is valid, it is not recommended. This may cause problems for
-    // systems that do not use start_time to determine when the aggregation
-    // value was reset (e.g. Prometheus).
-    AGGREGATION_TEMPORALITY_CUMULATIVE = 2;
-  }
+// Gauge represents the type of a scalar metric that always exports the
+// "current value" for every data point. It should be used for an "unknown"
+// aggregation.
+// 
+// A Gauge does not support different aggregation temporalities. Given the
+// aggregation is unknown, points cannot be combined using the same
+// aggregation, regardless of aggregation temporalities. Therefore,
+// AggregationTemporality is not included. Consequently, this also means
+// "StartTimeUnixNano" is ignored for all data points.
+//
+// A Metric of this Type MUST store its values as Int64DataPoint or
+// DoubleDataPoint.
+message Gauge {
+  repeated Int64DataPoint int64_data_points = 1;
+  repeated DoubleDataPoint double_data_points = 2;
+
+  // It describes the value type of the measurement used to build this
+  // aggregation.
+  //
+  // Determines if the points are Int64DataPoint or DoubleDataPoint, as well
+  // as the value type of the exemplars.
+  MeasurementValueType measurement_value_type = 3;
+}
+
+// Sum represents the type of a numeric scalar metric that is calculated as a
+// sum of all reported measurements over a time interval.
+//
+// A Metric of this Type MUST store its values as Int64DataPoint or
+// DoubleDataPoint.
+message Sum {
+  repeated Int64DataPoint int64_data_points = 1;
+  repeated DoubleDataPoint double_data_points = 2;
+
+  // It describes the value type of the measurement used to build this
```

Comment:
I don't think it's for this PR, but I notice there are three patterns for comments

- Go-style, repeating name in javadoc e.g. Sum represents the type
- Starting with `It`, otherwise similar to go-style
- Java-style, documenting behavior directly e.g. Represents the type of a metric

Should this be made consistent? I have a personal preference to Java-style for protos since I've seen that more in the past, but it looks like we use Go-style more here so consolidating on that would make sense too.

Reply:
Please file an issue so we don't forget about this